ranges_file.txt

```diff
@@ -0,0 +1,194 @@
+5389571-5389577
```

Comment:
Do we need to keep it ?

And for all replay json, if they all crashed the same error, maybe keep only one?

Reply:
We don't need this file, I will delete it.

Reply:
I will delete it later, it's something I discussed with @gusiri (the ability to provide custom `ranges.csv` files)